tests/Rector/ClassMethod/AddRouteAnnotationRector/Fixture/fixture_missing_route.php.inc

```diff
@@ -26,7 +26,7 @@ use Symfony\Bundle\FrameworkBundle\Controller\Controller;
 class AppController extends Controller
 {
     /**
-     * @\Symfony\Component\Routing\Annotation\Route(path="/all/{foo}/{page}", name="all", defaults={"foo" = "foo123", "page" = 1}, host=m.example.com, schemes={"http", "https"}, methods={"GET", "POST"}, condition="context.getMethod() in ['GET', 'HEAD'] and request.headers.get('User-Agent') matches '/firefox/i'", requirements={"page" = "\d+"}, options={"compiler_class" = "App\Routing\Utf8RouteCompiler", "expose" = "1", "foo1" = 5, "foo2" = "bar"})
```

Comment:
Could you add this as a new test fixture? The null case might repeat for other annotations too.

Reply:
Do you mean like this? https://github.com/rectorphp/rector-symfony/pull/246/files#diff-70343deb30a79daad564ef30ec87ee6db89ea53dc10eb30e988b88b3985636bb

Reply:
Yes, exactly :+1: 

Thanks